python/tskit/drawing.py

```diff
@@ -229,8 +233,12 @@ def draw(self):
                 labels.add(dwg.text(self._node_labels[u], (x[0] + dx, x[1] + dy)))
             if self._tree.parent(u) != NULL_NODE:
                 y = self._x_coords[v], self._y_coords[v]
-                lines.add(dwg.line(x, (x[0], y[1])))
-                lines.add(dwg.line((x[0], y[1]), y))
+                if self._edge_colours.get(u, None) is not None:
```

Comment:
What does ``stroke=None`` do here if we let it go through? It would be nice to avoid the condition here, no point in doing fiddly drawing code twice.

Reply:
I'm trying to avoid producing an SVG with looks of stroke="black" commands as the default. I could code it up to create a dict `params = {}` or `params = {'stroke':'black'}`, then call `dwg.line(..., **params)` only once, if you prefer.